### PR TITLE
Incorporating some of the feedback into text

### PR DESCRIPTION
**Pull request by mhalmagiu**
Incorporating some of the feedback into the text

### CODE REVIEW

File: src/content/11/en/part11a.md [hunk 1]
```diff
@@ -32,11 +54,23 @@ We'll discuss each of these steps (and when they're suitable) in more detail lat
...
 
+### Packaging and Deployment as a part of CI
+
+It may be worthwhile to note that packaging and especially deployment are sometimes considered to not fall under the umbrella of CI. We'll add them in here because in the real world it makes sense to lump it all together. This is partially because they make sense in the context of the flow and pipeline (I want to get my code to users) and partially because these are in fact the most likely point of failure.
+
+If a developer is working correctly in their own environment, it is unlikely that they will push code that would fail linting or tests. They would have already checked that on their own machine. The build too would likely pass because the tests would have likely had something to say about any issues that would prevent a build. It is possible that there will be a failure related to code in the master branch from other developer's changes and finding these issues is arguably the most important job of CI.
```
Comment (anna-sunberg) [inline src/content/11/en/part11a.md]: I don't really follow this paragraph. You say that it's _unlikely_ broken code will be pushed, but the last sentence says it's the most important job (why, if it's unlikely) and mentions some "other developer".

Comment (mhalmagiu) [reply src/content/11/en/part11a.md]: Fair point, let's see if I can fix that.

Comment (mhalmagiu) [reply src/content/11/en/part11a.md]: @anna-osipova I removed that paragraph and made a few changes to the next one. Does it make more sense now?

File: src/content/11/en/part11a.md [hunk 2]
```diff
@@ -15,11 +15,33 @@ If we're both working on changes and we haven't decided on a way to deploy to pr
...
+
+This step is much more complicated (and required) in compiled languages where the code needs to be compiled into an executable.
+
+#### Deploy
+Deployment refers to putting the software where it needs to be for the end user to use it. In the case of libraries, this may simply mean pushing an npm package to a package archive (such as npmjs.com) where other users can find it and include it in their software. 
+
+Deploying a service (such as a web app) may be a little more complicated. In this module we'll cover a simple deployment to a web based hosting service where users can use the software simply by going to the URL. We won't cover it in this module but it's important to know that deployments can be significantly more complex especially if we add requirements such as "the software must be available at all times during the deployment" (zero downtime deployments) or if we have to take things like database migrations into account.
```
Comment (TomiHiltunen) [inline src/content/11/en/part11a.md]: > Deploying a service (such as a web app) may be a little more complicated. In this module we'll cover a simple deployment to a web based hosting service where users can use the software simply by going to the URL. We won't cover it in this module but it's important...

This part took me a few read-throughs. We should cover basic deployment of a service or such. We don't need to go all-in on it by taking in also things like zero downtime deployments which could be considered a more advanced/refined deployment procedure.

**Review by TomiHiltunen — COMMENTED** [review]
Generally I think this is a great improvement. Just left one comment!

**Review by TomiHiltunen — APPROVED** [review]

**Pull request merged**